### DISCUSSION TITLE
Some downloaded programs won't run on my Windows 11 Virtual machine

### DISCUSSION THREAD

**Discussion by Proofeture** [utmapp/UTM#6151] [Windows]
Downloaded chrome, runs no problem. dowloaded a CAD software to use (MSI file) it installed no problem but when ever I try to run it, it loads for a second and just never opens. I am running a windows 11 virtual machine through UTM on mac m1

virtual machine
4.4.4
apple silicon
macbook pro
Through UTM

**Comment by Rastafabisch** [comment]
Especially for CAD software this is most likely due to the lack of GPU acceleration within UTM (qemu). It might also be related to windows x64 emulation though.

if it’s due to the latter one you’re out of luck. In case of the first one I’d recommend trying [VMWare Fusion](https://www.vmware.com/products/fusion.html) which is [free to use for personal use](https://vmware.com/go/get-fusionplayer) and features GPU acceleration for windows clients.
